app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -21,11 +22,15 @@ def update
       )
 
       if result.success?
-        if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
+        if how_to_verify_form_params['selection'] == 'remote'
+          idv_session.skip_doc_auth = false
           redirect_to idv_hybrid_handoff_url
         else
+          idv_session.flow_path = 'standard'
```

Comment:
Trying to treat my being new to this section as an asset: I notice we set `flow_path` here but not in the above condition(lines 25-26). Is that fine and it's just that we don't need to update it?

Reply:
idv_session.flow_path is set by HybridHandoff (to either 'standard' or 'hybrid') so you only want to set it if you're skipping HybridHandoff and going directly to DocumentCapture.